### PR TITLE
Optionally set nginx_version

### PR DESCRIPTION
**Pull request by manics**
New feature, tag `1.1.0`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Role Variables
 --------------
 
 - `nginx_keep_default_configs`: If `true` keep the default site configuration files in `nginx/conf.d`, default `false` (disable them)
+- `nginx_version`: The packaged version of Nginx, optional
```
Comment (sbesson) [inline README.md]: Is this logic independent of the value of `nginx_stable_repo`?

Comment (sbesson) [inline README.md]: Is this logic dependent of the value of `nginx_stable_repo`?

Comment (manics) [reply README.md]: No, you can only install versions that are available in the repo.

Comment (sbesson) [reply README.md]: :+1: maybe we want to group both option together and indicate the relationship?

**Comment by sbesson** [comment]
Barring the README comment in https://github.com/openmicroscopy/ansible-role-nginx/pull/3#discussion_r266429476, changes look good and the new logic is tested by Molecule. I assume the next step is to test it on a production/test server? Happy to see this merged and tagged.

File: README.md (hunk 1 above)
Comment (manics) [reply README.md]: updated

**Pull request merged**